.github/workflows/build.yml

```diff
@@ -170,7 +170,7 @@ jobs:
         run: |
           opam install lwt logs # Also build pet-server
           opam install memprof-limits # We need to do this to avoid coq-lsp rebuilding Coq below due to deptops
-          opam install vendor/coq/{coq-core,rocq-core,stdlib/coq-stdlib,coqide-server,coq}.opam
+          opam install vendor/coq/{rocq-runtime,rocq-core,stdlib/coq-stdlib,coqide-server,coq}.opam
```

Comment:
stdlib still depends on coq-core until a followup PR
and coq will depend on coq-core forever (but we will add a `rocq` package ofc)
so just removing coq-core from the "to install" list probably won't work

Reply:
Rest is working fine, waiting for the web worker artifact to test it.